adcircpy/forcing/winds/best_track.py

```diff
@@ -276,7 +286,10 @@ def storm_id(self, storm_id: str):
             digits = sum([1 for character in storm_id if character.isdigit()])
 
             if digits == 4:
-                atcf_id = get_atcf_id(storm_name=storm_id[:-4], year=int(storm_id[-4:]))
+                try:
```

Comment:
Yea basically only comment is to avoid bare excepts. Perhaps you could make the "check if you have internet" thing as a class method like this 
https://www.kite.com/python/answers/how-to-check-internet-connection-in-python

Reply:
updated with 26ec0bd